gno.land/cmd/gnoland/testdata/event.txtar

```diff
@@ -0,0 +1,42 @@
+# start a new node
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/ee -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+
+gnokey maketx call -pkgpath gno.land/r/demo/ee -func Hello -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1
+stdout OK!
+stdout 'GAS WANTED: 2000000'
+stdout 'GAS USED: [0-9]+'
+stdout 'EVENTS:'
+
+-- ee.gno --
+package ee
+
+import (
+	"std"
+)
+
+const (
+	EventAttrKeySender = "sender"
+	EventAttrKeyReceiver = "receiver"
+)
+
+func Hello() {
+	std.Emit(
+		EventAttrKeySender,
+		[]string{
+			"key1", "value1",
+			"key2", "value2",
+			"key3", "value3",
+			"not-a-pair",		// ignore the key-value not in pairs
```

Comment:
should we ignore? might be better to make it a key, with empty value, so at least the developer can see that it exists with no value. also allows a single attribute to be set, and still show up.

Reply:
Just like the ellipsis case, if it were possible to pass an `interface` type as a parameter (to use a struct directly), it would have been set to an empty string value. But since only arrays are currently usable, there's a chance of unintended omissions of keys or values in the middle.

I think this could lead to unintended behaviors, so the values were set to be ignored when a remainder exists. I believe there's better way to handle this, but I can't think of it now.